src/diffpy/pdfgui/gui/phasenotebookpanel.py

```diff
@@ -40,7 +40,11 @@ def __init__(self, *args, **kwds):
         self.notebook_phase_pane_Configure   = PhaseConfigurePanel(self.notebook_phase, -1)
         self.notebook_phase_pane_Constraints = PhaseConstraintsPanel(self.notebook_phase, -1)
         self.notebook_phase_pane_Results     = PhaseResultsPanel(self.notebook_phase, -1)
+        self.notebook_phase_pane_Configure.notebook_phase = self.notebook_phase
 
+        self.notebook_phase_pane_Configure.notebook_phase_pane_Configure = self.notebook_phase_pane_Configure
```

Comment:
can you change these lines into the style like `PhaseConfigurePanel(self.notebook_phase, -1)`?